### PR TITLE
Editorial pass

### PR DESCRIPTION
**Pull request by anssiko**
- Update spec names
- Editorial nits, remove trailing "." from list items
- Update Charter history

PTAL @dontcallmedom

### CODE REVIEW

**Comment by anssiko** [comment]
Proposed one more change https://github.com/w3c/dap-charter/pull/39/commits/0ca66764a17e1a3da2bcef72be8802ff82a25bc1, while not strictly editorial, rolled it into this branch for convenience.

File: DeviceAPICharter.html [hunk 1]
```diff
@@ -48,7 +48,7 @@
 
 <h1 id="title">[DRAFT] Device and Sensors Working Group Charter</h1>
 
-<p><em>This is a draft revised Device APIs Working Group (renamed Device and Sensors Working Group) charter for discussion. It has no standing. See also the <a href="https://www.w3.org/2016/03/device-sensors-wg-charter.html">charter under which the Device APIs Working Group</a> currently operates.</em></p>
+<p><em>This is a draft revised Device APIs Working Group (renamed Device and Sensors Working Group) charter for discussion. It has no standing. See also the <a href="https://www.w3.org/2016/03/device-sensors-wg-charter.html">charter under which the Device and Sensors Working Group</a> currently operates.</em></p>
```
Comment (dontcallmedom) [inline DeviceAPICharter.html]: maybe flip the first: Device APIs Working Group (renamed Device and Sensors Working Group) -> Device and Sensors Working Group (formerly known as Device APIs Working Group); or drop the mention of Device APIs completely?

**Review by dontcallmedom — APPROVED** [review]
suggesting an additional fix, but otherwise lgtm

File: DeviceAPICharter.html (hunk 1 above)
Comment (anssiko) [reply DeviceAPICharter.html]: Proposed fix https://github.com/w3c/dap-charter/pull/39/commits/69ec9e9214f4e035aaae884995cae4c03a7875c9

**Comment by anssiko** [comment]
@dontcallmedom, I pushed https://github.com/w3c/dap-charter/pull/39/commits/b265f79e7ab55c25a7132f4fc36c18f21b0130a2 to this branch since your previous comment, PTAL.

**Comment by dontcallmedom** [comment]
still LGTM :)

**Comment by anssiko** [comment]
Feel free to merge. If I find other editorial changes I can open another PR.

(btw. what is the DL to get editorial changes in?)

**Comment by dontcallmedom** [comment]
it would be good to get all changes in before next Tuesday, so that we can bring this to W3C management review next Wednesday

**Pull request merged**